tests/gha-run-k8s-common.sh

```diff
@@ -21,6 +20,12 @@ function _print_cluster_name() {
     echo "${test_type}-${GH_PR_NUMBER}-${short_sha}-${KATA_HYPERVISOR}-${KATA_HOST_OS}-amd64"
 }
 
+function _print_rg_name() {
+    test_type="${1:-k8s}"
+
+    echo "kataCI-$(_print_cluster_name ${test_type})"
```

Comment:
As part of the cluster name, we'll have to add "small" "normal", otherwise I think we may have some clashes.

Reply:
Either in the rg name or in the cluster name, and I only noticed this after my PR got merged, sorry.